pkg/services/librarypanels/database.go

```diff
@@ -28,12 +48,17 @@ func (lps *LibraryPanelService) createLibraryPanel(c *models.ReqContext, cmd cre
 		UpdatedBy: c.SignedInUser.UserId,
 	}
 	err := lps.SQLStore.WithTransactionalDbSession(context.Background(), func(session *sqlstore.DBSession) error {
+		err := setLibraryPanelHeader(&libraryPanel)
```

Comment:
Can you move the `setLibraryPanel` call out of this database callback? I don't see any reason for having it there. This change  will make the code much easier to read and verify.

Reply:
Yeah, sure I think I can do that.

Reply:
Fixed!